content/pages/privacy/en.md

```diff
@@ -162,92 +163,86 @@ Our legitimate interest is to answer your inquiries.
 
-If you published a data asset, please note that you can modify your provided metadata (like the author&#39;s name) at any time by conducting the following steps.
+If you published a data service, please note that you can modify your provided metadata at any time by conducting the following steps.
 
-- Visit your published asset on our market ([portal.minimal-gaia-x.eu](https://portal.minimal-gaia-x.eu)).
+- Visit your published data service on our portal demonstrator ([tba.lab.gaia-x.eu](https://tba.lab.gaia-x.eu)).
 - Validate your identity with your private key.
 - Click &quot;EDIT METADATA&quot; and make your changes.
 - Save your changes by clicking &quot;SUBMIT&quot;.
 - Confirm the changes with your private key.
 
-Please note that you have to pay gas fees for the confirmation of the changes (as a new transaction is issued). After alteration, the previous version of your data asset metadata will no longer be visible on our market. Also, you can change the content of your data asset at any time.
+Please note that you have to pay gas fees for the confirmation of the changes (as a new transaction is issued). After alteration, the previous version of your data service metadata will no longer be visible on our portal demonstrator. Also, you can change the content of your data service at any time.
 
 ## 7.3 Right to erasure (Art. 17 GDPR)
 
 You have the right to obtain without undue delay the erasure of personal data about you, where the defined legal grounds in Art. 17 GDPR apply.
 
-If you published a data asset, you could overwrite your provided author name with a placeholder by conducting the steps listed in chapter 7.2. If you want to disable displaying your data asset on our market, please read chapter 7.4.
+If you published a data service, you could overwrite your provided metadata with a placeholder by conducting the steps listed in chapter 7.2. If you want to disable displaying your data service on our portal demonstrator, please read chapter 7.4.
 
 Note that it is not possible to erase the blockchain&#39;s transaction history due to technical reasons. But the current network state will no longer hold the former metadata.
 
-Moreover, due to technical reasons, you cannot erase your provided wallet address. The permanent storage of the wallet address protects you as an asset publisher as well as an asset consumer. Data publishers can prove asset ownership and asset sales, while data consumers can prove asset acquisition and consumption.
+Moreover, due to technical reasons, you cannot erase your provided wallet address. The permanent storage of the wallet address protects you as a data services publisher as well as an data service consumer. Data services publishers can prove data service ownership and sales, while data service consumers can prove data service acquisition and consumption.
```

Comment:
issue(blocking): Change `publisher as well as an data service consumer. Data services publishers...` to `publisher as well as a data service consumer. Data service publishers...`